casper/src/main/scala/coop/rchain/casper/engine/LfsTupleSpaceRequester.scala

```diff
@@ -35,9 +35,10 @@ object LfsTupleSpaceRequester {
     */
   type StatePartPath = Seq[(Blake2b256Hash, Option[Byte])]
 
-  // TODO: extract this as a parameter
-  // 20,000 records uses about 2G of direct buffer memory on importing side
-  val pageSize = 20000
+  /**
+    * Number of nodes in LFS sync data transfer
+    */
+  val pageSize = 750
```

Comment:
is this change here on purpose? it seems unrelated.

Reply:
@TheoXD Can you rebase as suggested by @tgrospic? https://github.com/rchain/rchain/pull/3753#issuecomment-1162860649

Reply:
We are not squashing commits on merge so intermediate changes on PR should be removed.

Reply:
New PR submitted without intermediate changes: #3777